core/system/src/unvic.c

```diff
@@ -94,7 +94,7 @@ static void unvic_acl_check(int irqn)
 void unvic_isr_set(uint32_t irqn, uint32_t vector, uint32_t flag)
 {
     /* verify IRQ access privileges */
-    unvic_acl_check(irqn);
+    unvic_acl_check(irqn, g_active_box);
```

Comment:
Because although most of the time it is `g_active_box`, there are special cases in which it is not. An important example: cross-box IRQ handling (see https://github.com/ARMmbed/uvisor/pull/100/files#diff-8079bf2b7ed17a83b95fadc1b10f7625R268), where `dst_id` is used instead. It is related to https://github.com/ARMmbed/uvisor/issues/99